plugins/terminal.koplugin/profile

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+# This file gets executed on every start of the KOTerm shell.
+# Do not edit this file!
+# You may change the contents of 'aliases' and 'profile.user'
+#
+
+KOTERM_HOME="$(pwd)"
+# make KOTERM_DATA an absolute path
+# shellcheck disable=SC2164
+cd "${KOTERM_DATA}"
```

Comment:
What it's saying is that all the following stuff will be doing unexpected things if you this fails and you continue the script, for which it almost certainly suggests this as the default solution:
```suggestion
cd "${KOTERM_DATA}" || exit
```